packages/backend/src/core/queue/TaskQueue.ts

```diff
@@ -206,4 +206,16 @@ export class TaskQueue<T> {
       setTimeout(() => this.execute())
     }
   }
+
+  // WARNING: this method clears the queue, be cautious when using it
+  // some Updaters will not function properly after you unhalt them using this method
+  // because they rely on the start() function which is called only once
+  // so use it only in Updaters with generic (updating all the missing data) update() function
+  // or rewrite the logic of your updater
+  unhaltIfNeeded() {
+    if (this.halted) {
+      this.queue.splice(0, this.queue.length)
```

Comment:
```suggestion
      this.queue = []
```
Why not like this?

Reply:
![image](https://github.com/l2beat/l2beat/assets/72789647/923a9270-408f-43b3-b3e8-710d71002a70)